api/src/data_migration/transformation/transform_oracle_data_task.py

```diff
@@ -840,21 +860,26 @@ def process_link_funding_instrument(
         target_funding_instrument: LinkOpportunitySummaryFundingInstrument | None,
         opportunity_summary: OpportunitySummary | None,
     ) -> None:
-        self.increment(self.Metrics.TOTAL_RECORDS_PROCESSED)
+        self.increment(self.Metrics.TOTAL_RECORDS_PROCESSED, prefix=FUNDING_INSTRUMENT)
         extra = transform_util.get_log_extra_funding_instrument(source_funding_instrument)
         logger.info("Processing funding instrument", extra=extra)
 
+        if source_funding_instrument.is_deleted:
```

Comment:
Is there a specific reason you're moving this case higher in the if elif statements?

Reply:
The `_handle_delete` function handles the case where something simultaneously hits both the `is_deleted` and orphaned record case at once.

In reality the if/elif-statements are:
* is deleted + orphaned (inside `_handle_delete`)
* is deleted (inside `_handle_delete`)
* orphaned + historical
* insert